lib/sles4sap_publiccloud.pm

```diff
@@ -346,7 +378,8 @@ sub stop_hana {
         record_info("Wait ssh disappear start");
         my $out = $self->{my_instance}->wait_for_ssh(timeout => 60, wait_stop => 1);
```

Comment:
in a next PR we should try to remove this one too. We are using this `wait_for_ssh` in a way, for a purpose, is not supposed to be used: to detect that the VM is down.